domain-model/src/main/java/uk/gov/hmcts/cmc/domain/amount/TotalAmountCalculator.java

```diff
@@ -29,15 +29,22 @@ private TotalAmountCalculator() {
     }
 
     public static Optional<BigDecimal> amountWithInterest(Claim claim) {
-        return Optional.ofNullable(calculateTotalAmount(claim, LocalDate.now(), false));
+        LocalDate date = claim.getCountyCourtJudgmentRequestedAt() == null ? LocalDate.now()
```

Comment:
Can these lines be extracted out as single private static method and reused between two functions?